students/arkadiusz_kasprzyk/lesson_01_basics/clock_face_1.py

```diff
@@ -0,0 +1,39 @@
+'''
+title: clock_face_1
+author: arkadiusz.kasprzyk@tieto.com
+date: 2018-03-05
+description:
+    H hours, M minutes and S seconds are passed since the midnight (0 ≤ H < 12, 0 ≤ M < 60, 0 ≤ S < 60).
+    Determine the angle (in degrees) of the hour hand on the clock face right now.
+'''
+
+import math as m
+print('''
+    H hours, M minutes and S seconds are passed since the midnight (0 ≤ H < 12, 0 ≤ M < 60, 0 ≤ S < 60). 
+    Determines the angle (in degrees) of the hour hand on the clock face right now.
+''')
+
+H = int(input("hours: "))
+M = int(input("minutes: "))
+S = int(input("seconds: "))
+
+deg_per_hour = 30  ## = 360/12
+min_per_minute = 30 ## 1/2  deg_per_minute = 30/60
+sec_per_second = 30 ## 1/2  min per second = 30/60
+
+degs = H * deg_per_hour
+mins = M * min_per_minute
+secs = S * sec_per_second
+
+
+mins = mins + secs // 60
+secs = secs % 60
+
+degs = degs + mins // 60
+mins = mins % 60
+
+
+print("Angle (in deg.min.sec) of the hour hand is {}° {}' {}''".format(degs, mins, secs))
```

Comment:
It's really not clear what they mean. I've chosen more traditional way of noting angles (and little more chalenging) and got rapped... After all it works!

Reply:
I haven't looked at test cases - it could have spoiled the fun of solving problem.

Reply:
Like I wrote in the introduction it's not personal. It's mostly about Python and program engineering. So from my experience in real world:

You get the requirements in written form or as unit tests (which is really great, but rarely happens) and your task is to pass, because that's the way to cooperate with other people on the project. Most of the time the requirements are unclear and you have to ask, you should rather be happy, if you get unit tests beforehand.

Also, most of the time the part you write is not a standalone solution, but rather a part of a solution and any superfluous input handling, prints or format mismatch will not be appreciated. To me it's "egal", because it's not real world. I'm just trying to pass that knowledge on.